### PR TITLE
Get store admin and store assistant analytics from api

### PR DESCRIPTION
**Pull request by kofimokome**
# Description

1. Store assistant dashboard now gets data from the api
2. Store assistant dashboard now gets data from the api

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: resources/views/backend/assistant/show.blade.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
                     <a href="{{ route('assistants.index') }}" class="btn btn-primary float-right">
                         Go Back
                     </a>
-                    <a href="{{route('assistants.edit', $assistant->_id) }}" class="mr-3 btn btn-success float-right">
+                    <a href="{{route('assistants.edit', $data->_id) }}" class="mr-3 btn btn-success float-right">
```
Comment (Jeremiahiro) [inline resources/views/backend/assistant/show.blade.php:24]: why did you change from assistant to data?

File: resources/views/partials/dashboard/store_assistant.blade.php [hunk 2]
```diff
@@ -21,19 +21,54 @@
...
+                                            <div class="modal-footer">
+                                                <form action="{{ route('assistants.destroy', $data->_id) }}"
+                                                      method="POST" id="form">
+                                                    @method('DELETE')
+                                                    @csrf
+                                                    <button type="submit" class="btn btn-primary btn-danger">
+                                                        Delete
+                                                    </button>
+                                                </form>
+                                                <button type="button" class="btn btn-secondary" data-dismiss="modal">No,
```
Comment (Jeremiahiro) [inline resources/views/partials/dashboard/store_assistant.blade.php:64]: the text does not add up

File: resources/views/backend/assistant/show.blade.php (hunk 1 above)
Comment (kofimokome) [reply resources/views/backend/assistant/show.blade.php:24]: compatibility issues.
I used this same file in Dashboard view and single assistant view
